src/docs/api/public.mdx

```diff
@@ -248,4 +366,5 @@ No problem. Send us an [email](mailto:partners@sentry.io) so we can answer your
 
 ## Can customers use private endpoints?
```

Comment:
This is bigger than the rest of the questions in the vercel preview 🚼

Reply:
I think that's intentional because it's an important caveat we want to highlight